### DISCUSSION TITLE
How to get the Galaxy Brain of Github?

### DISCUSSION THREAD

**Discussion by solidm568** [community/community#56568] [New to GitHub] [answered]
How do I get the Galaxy Brain badge? Please help me

**Comment by SolidMaster568** [comment, marked as answer]
To achieve the 'Galaxy Brain' achievement, take note of the following:
1. Answer at least 2 discussions.
2. Take note, you CANNOT mark your own answers to your own questions. They have to be marked by another user.
3. Discussions should be in a public repository.
4. There's a current bug observed by some users whereby marking your own answers can prevent you from getting the badge instead.

Hope this answer has helped you, and all the best in getting the achievement.

**Reply by solidm568** [reply]
Thanks for your reply.
It worked. really... :)

**Comment by queenofcorgis** [comment]
@solidm568 @MX56893 GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answering and locking the post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
